### PR TITLE
Ports Baystation's handling of monkey/human NPCs.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Also ports monkey sprites.
Removes leftover resomi sprites.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

File: code/_helpers/turfs.dm [hunk 1]
```diff
@@ -25,6 +25,20 @@
...
 	return 1
 
+/proc/get_random_turf_in_range(var/atom/origin, var/outer_range, var/inner_range)
+	origin = get_turf(origin)
+	if(!origin)
+		return
+	var/list/turfs = list()
+	for(var/turf/T in orange(origin, outer_range))
+		if(!(T.z in current_map.sealed_levels)) // Picking a turf outside the map edge isn't recommended
+			if(T.x >= world.maxx-TRANSITIONEDGE || T.x <= TRANSITIONEDGE)	continue
```
Comment (skull132) [inline code/_helpers/turfs.dm]:
```suggestion
			if(T.x >= world.maxx-TRANSITIONEDGE || T.x <= TRANSITIONEDGE)
				continue
```

File: code/_helpers/turfs.dm [hunk 2]
```diff
@@ -25,6 +25,20 @@
...
+			if(T.y >= world.maxy-TRANSITIONEDGE || T.y <= TRANSITIONEDGE)	continue
```
Comment (skull132) [inline code/_helpers/turfs.dm]:
```suggestion
			if(T.y >= world.maxy-TRANSITIONEDGE || T.y <= TRANSITIONEDGE)
				continue
```

File: code/modules/mob/living/carbon/human/species/station/human/human.dm [hunk 3]
```diff
@@ -41,4 +41,48 @@
...
+		H.emote("scream")
+
+	if(!H.restrained() && H.lying && H.shock_stage >= 60 && prob(3))
+		H.custom_emote("thrashes in agony")
+
+	if(!H.restrained() && H.shock_stage < 40 && prob(3))
+		var/maxdam = 0
+		var/obj/item/organ/external/damaged_organ = null
+		for(var/obj/item/organ/external/E in H.organs)
+			if(!E.can_feel_pain()) continue
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/species/station/human/human.dm]:
```suggestion
			if(!E.can_feel_pain())
				continue
```

File: code/modules/mob/living/carbon/human/species/station/human/human.dm [hunk 4]
```diff
@@ -41,4 +41,48 @@
...
+				H.custom_emote("clutches [T.his] [damaged_organ.name], trying to stop the blood.")
+			else if(damaged_organ.status & ORGAN_BROKEN)
+				H.custom_emote("holds [T.his] [damaged_organ.name] carefully.")
+			else if(damaged_organ.burn_dam > damaged_organ.brute_dam && damaged_organ.organ_tag != BP_HEAD)
+				H.custom_emote("blows on [T.his] [damaged_organ.name] carefully.")
+			else
+				H.custom_emote("rubs [T.his] [damaged_organ.name] carefully.")
+
+		for(var/obj/item/organ/I in H.internal_organs)
+			if((I.status & ORGAN_DEAD) || BP_IS_ROBOTIC(I)) continue
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/species/station/human/human.dm]:
```suggestion
			if((I.status & ORGAN_DEAD) || BP_IS_ROBOTIC(I))
				continue
```

File: code/modules/mob/living/carbon/human/species/station/human/human.dm [hunk 5]
```diff
@@ -41,4 +41,48 @@
...
+			if(I.damage > 2) if(prob(2))
+				var/obj/item/organ/external/parent = H.get_organ(I.parent_organ)
+				H.custom_emote("clutches [T.his] [parent.name]!")
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/species/station/human/human.dm]: no. indent further.

**Comment by ToasterStrudes** [comment]
Its 2020 and were still removing resomi

**Comment by KasparoVy** [comment]
Can never remove them from my heart, rip too cute for this world

**Pull request merged**